lib/PuppeteerSharp/Launcher.cs

```diff
@@ -215,6 +219,7 @@ private void CleanUp()
             if (_chromeProcess.Id != 0 && Process.GetProcessById(_chromeProcess.Id) != null)
             {
                 _chromeProcess.Kill();
+                _chromeProcess.WaitForExit();
```

Comment:
Could you talk a little bit about this?
I'm working on these lines now and I'd like to know what you've discovered so far.

Reply:
I had an issue with deleting the chrome profile directory in the test, I was receiving an access denied error. It looked like the `Kill` method had a small delay before completing

Reply:
That's why I commented out the delete directory line in the tests